addon/index.js

```diff
@@ -0,0 +1,41 @@
+import Rollbar from "rollbar";
+import Ember from 'ember';
+
+const rollbar = new Rollbar({
+  enabled: false,
+  captureUnhandledRejections: typeof FastBoot === 'undefined'
+});
+
+let oldOnError;
+
+function startRollbar(config = {}) {
+  rollbar.configure({...{ enabled: true }, ...config, ...{
+    captureUnhandledRejections: typeof FastBoot === 'undefined'
+  }});
+
+  oldOnError = Ember.onerror
+
+  Ember.onerror = (...args) => {
+    if (rollbar) rollbar.error(args[0]);
+
+    if (typeof oldOnError === 'function') {
+      oldOnError(...args);
+    } else if (!rollbar.options.enabled ||   Ember.testing) {
+      throw args[0];
+    }
+  };
+
+  return rollbar;
+}
+
+function stopRollbar() {
+  rollbar.options.enabled = false;
+  Ember.onerror = oldOnError;
+}
+
+export {
+  rollbar,
+  Rollbar,
+  startRollbar,
+  stopRollbar
+}
```

Comment:
```suggestion
```

Reply:
I prefer this way of exporting things in modules. I help me clearly determine what is public interface by just scrolling to the end of the file.

Reply:
Ah.. I think I'm conflating two different things.. Looks like they are still named exports, just as a list? That's ok then 😅 

For reference https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export